.changeset/unlucky-onions-reply.md

```diff
@@ -0,0 +1,7 @@
+---
+"@wso2is/console": patch
+"@wso2is/identity-apps-core": patch
+"@wso2is/i18n": patch
```

Comment:
Let's patch the myaccount as well to keep the i18n dependency version consistent. 
PS: Need to find a way to enforce this :)

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/4280/commits/31b88ad29860876b65b8843de775b5b7a096cd5e